website/content/docs/fork-workflow.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: Enable External Contributors with the Fork Workflow
+title: Fork Workflow
 group: features
 ---
 
-When using the GitHub backend, you can use Netlify CMS to accept contributions from GitHub users without giving them access to your repository. When they make changes in the CMS, the CMS forks your repository for them behind the scenes, and all the changes are made to the fork. When the contributor is ready to submit their changes they can submit a pull request to your repository, which you can merge using the GitHub UI. At the same time, any contributors who _do_ have write access to the repository can continue to use the CMS normally.
+When using the [GitHub backend](/docs/authentication-backends/#github-backend), you can use Netlify CMS to accept contributions from GitHub users without giving them access to your repository. When they make changes in the CMS, the CMS forks your repository for them behind the scenes, and all the changes are made to the fork. When the contributor is ready to submit their changes, they can set their draft as ready for review in the CMS. This triggers a pull request to your repository, which you can merge using the GitHub UI.
```

Comment:
Doesn't show in the diff, but I made these changes:

- Added a link for GitHub backend I tend to err on the side of linking too much, and this is the pattern in most of the docs. I like having shortcuts, but feel free to remove if it feels too distracting.
- Edited the last couple of sentences to emphasize that the editor submits the PR via the CMS, and doesn't have to go to GitHub.